Dockerfile

```diff
@@ -1,11 +1,20 @@
 # Build the manager binary
-FROM golang:1.10.3 as builder
+FROM golang:1.11.4 as builder
+
+ENV GO111MODULE=on
 
 # Copy in the go src
 WORKDIR /go/src/github.com/kubeflow/xgboost-operator
+
+COPY go.mod .
+COPY go.sum .
+# COPY vendor/ vendor/
+
+RUN go mod download
```

Comment:
What's the purpose of this change? Is this being used somewhere?

Reply:
we need to setup the go build env based on the go.sum and go.mode. or else, the docker build would break.